### PR TITLE
Manuscript reorganization recommendations

### PR DESCRIPTION
**Pull request by WesIngwersen**
This draft has a lot of useful content, but some heavy reorganization and recasting of the information in it to create more of a story is needed. 

Some of these additions would also be helpful

- Code stats 
- Code review status (perhaps in Result summary)
- Presentation of some of the standard formats- i think this can be in results.
 
I feel like there are 2 years of work that are not yet fairly represented. There should be something more more about some of the challenges that were faced and how they were overcome or if not as a story, at least some more depth on what ended up being put done, going into depth in the package, and why that was needed.

### CODE REVIEW

File: documentation/flowsaAppliedSciencesManuscript2021.md [hunk 1]
```diff
@@ -27,13 +27,14 @@ material flows, resource use by industry, water use by industry, waste by indust
...
+Calculating the direct use of raw materials and emissions generated by economic sectors is a complex and evolving process.
+Data is often provided in aggregated formats for broadly defined categories, and tools to assess environmental resource use can quickly become outdated when sources publish new data. 
 
 The USEPA created the Flow Sector Attribution (FLOWSA) Python tool to advance understanding and documentation of direct resource use and enable timely dataset updates when new source data is available. 
 FLOWSA is a data processing library developed to attribute physical movements of resources, wastes, losses, or emissions, generically called flows, to sectors [@birney_flowsa_2021]. 
 The term _flows_ is adopted from life cycle assessment terminology [@edelen_framework_2022]. 
 _Sectors_ are generally defined as economic sectors generating economic activity but are extended to include households and government end users.
-Sectors can be industries or commodities based on the Bureau of Economic Analysis (BEA) input-output (IO) table definitions.
+Sectors can be industries or commodities based on the Bureau of Economic Analysis (BEA) input-output (IO) table definitions.
```
Comment (MoLi7) [inline documentation/flowsaAppliedSciencesManuscript2021.md:37]: I thought `sectors` in flowsa are mostly, if not all, based on NAICS, aren't they?

File: documentation/flowsaAppliedSciencesManuscript2021.md [hunk 2]
```diff
@@ -135,7 +136,7 @@ import flowsa
 flowsa.seeAvailableFlowByModels(methodtype)
 ```
 Where _methodtype_ is either 'FBA' or 'FBS' dependent on the type of data a user would like to access. 
-To retrieve an FBA as a pandas data frame, a user can call on the customizable 'getFlowByActivity' function. 
+To retrieve an FBA as a pandas data frame, a user can call on the customizable 'getFlowByActivity' function.
```
Comment (MoLi7) [inline documentation/flowsaAppliedSciencesManuscript2021.md:139]: A ref to `pandas` would be helpful.

File: documentation/flowsaAppliedSciencesManuscript2021.md [hunk 3]
```diff
@@ -169,6 +170,7 @@ Developers can modify the existing Flow-By-Activity and Flow-By-Sector methodolo
 ### Generating Flow-By-Activity Datasets 
 The first step in the sector attribution models is formatting any required source data into a standardized Flow-By-Activity (FBA) table. 
 To generate an FBA, a user will need:
+
```
Comment (MoLi7) [inline documentation/flowsaAppliedSciencesManuscript2021.md:173]: Perhaps rephrase this to something more instructive, e.g. 'To generate an FBA, XX and XX are needed.', then suggest that, if users want to generate their own FBA datasets, customized config file in the format of the built-in method YAML is needed.
Similar rephrases to the FBS section below.

File: documentation/flowsaAppliedSciencesManuscript2021.md [hunk 4]
```diff
@@ -118,7 +118,7 @@ import appdirs
 appdirs.user_data_dir()
 ```
 The second option is to retrieve FLOWSA files for different package versions from the USEPA's Data Commons, an Amazon AWS s3 server (https://edap-ord-data-commons.s3.amazonaws.com/index.html?prefix=). 
-Storing files on a remote server enables convenient retrieval of these files by other LCA ecosystem tools, such as useeior, without requiring a user to install python. 
+Storing files on a remote server enables convenient retrieval of these files by other LCA ecosystem tools, such as useeior [@mo_li_useeior_2022], without requiring a user to install python.
```
Comment (MoLi7) [inline documentation/flowsaAppliedSciencesManuscript2021.md:121]: Several refs including this one is not available in FLOWSA.bib. Maybe consider using USEEIO.bib in USEEIO_team if many refs are shared between the two repos?

File: documentation/flowsaAppliedSciencesManuscript2021.md [hunk 5]
```diff
@@ -212,6 +212,21 @@ These datasets include the Toxic Release Inventory (TRI), National Emissions Inv
...
+yaml | 121 | 3746
+csv | 76 | 79047
+md | 14 | 340
+example | 1 | 2
+gitignore | 1 | 8
+ini | 1 | 3
+txt | 1 | 18
+yml | 1 | 47
+: FLOWSAv1.0.1 Code Statistics {#tbl:stats} 
+
```
Comment (catherinebirney) [inline documentation/flowsaAppliedSciencesManuscript2021.md:229]: @WesIngwersen thanks!

**Comment by catherinebirney** [comment]
Thanks @MoLi7 and @WesIngwersen for the abstract edits!

**Comment by WesIngwersen** [comment]
@catherinebirney 
The methods section is much improved.
The section _Sector Attribution Challenges_ needs to be moved to the Discussion. 
There are still some TODO lines and CITE instructions in the Methods.
Some of the refs are also missing from the bib file - see the pandoc output to see which are missing.
